scripts/gen-esm-proxy.mjs

```diff
@@ -0,0 +1,103 @@
+import { existsSync, mkdirSync, readFileSync, writeFileSync } from "node:fs";
```

Comment:
Maybe we should publish this as an npm package in the vein of `@bufbuild/license-header`? It seems like it could provide value to others too maybe.

Reply:
Not yet. Once the dust has settled, I'd like this thing to do a bit more work, so that we only need a single build script in all packages.